### ISSUE TITLE
Building Lithomop Dependency PetSc v2.3.3

### ISSUE THREAD

**Issue by hlokavarapu** [geodynamics/lithomop#1]
I am building Lithomop for CIG's continuous integration system called Jenkins. When attempting to build PetSc v2.3.3, the build errors out due to an unreachable server. Is there a solution to this problem? Is it possible to build Lithomop with a more recent version of PetSc? 

Jenkins - PetSc v2.3.3 [http://blofeld.geodynamics.org/job/PETSc_v2.3.3/](url)

Build Log - PetSc v2.3.3 [http://blofeld.geodynamics.org/job/PETSc_v2.3.3/Enable_MPI=1,tag_arch=64bit,tag_os=ubuntults/lastBuild/console](url)

**Comment by JarettBakerDunn** [comment]
Hello, I'm working on containerizing lithomop and I've been having trouble compiling lithomop as well. Do you recall if you were able to build petsc and lithomop? 

Thanks.

**Comment by ljhwang** [comment]
@knepley @jedbrown. Do you have any insights into this? Else we may have hit a deadend here.

**Comment by knepley** [comment]
Lithomop is not a supported product. The containerization should be cancelled.

**Comment by ljhwang** [comment]
Correct, Lithomop is an archived code. We currently have an effort to containerize, if possible active and archived code. Arguably this makes more sense for some codes in terms of FAIR, reuse and replicability and may not be possible for all codes. I would be curious to know whether anyone has built containers for older versions of PETSc

**Comment by JarettBakerDunn** [comment]
@ljhwang I'm not sure if this is what you're thinking of, but in the process of trying to build lithomop I have gotten PETSc 2.3.3 to compile in a Docker container. If it would be useful I could create a separate container which only contains PETSc.
